<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgDisReinvest</class>
 <widget class="QWidget" name="wdgDisReinvest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/xulpymoney.qrc">
    <normaloff>:/xulpymoney/tools-wizard.png</normaloff>:/xulpymoney/tools-wizard.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblTitulo">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 128, 0);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="radRe">
         <property name="text">
          <string>Reinvest simulation</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radDes">
         <property name="text">
          <string>Disinvest simulation</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Operation data</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="lblSimulacion"/>
            </item>
            <item>
             <widget class="myQLineEdit" name="txtSimulacion">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>1000</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="lblValor"/>
            </item>
            <item>
             <widget class="myQLineEdit" name="txtValorAccion">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Bank comission</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="myQLineEdit" name="txtComision">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>10</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Shares number</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtAcciones">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Amount</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtImporte">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmd">
       <property name="text">
        <string>Make simulation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>After</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QTabWidget" name="tabResultados_2">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="tab_9">
            <attribute name="title">
             <string>Investment operations</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="myQTableWidget" name="tblOperaciones">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Id</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Date</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Operation type</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Shares</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Price</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Balance</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Comission</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Taxes</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Situac_2">
            <attribute name="title">
             <string>Investment current state</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="myQTableWidget" name="tblInvestmentsActualDespues">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Date</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Shares</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Price</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Balance</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Pending</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>% Annual</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>% APR</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>% Total</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Benchmark</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_8">
            <attribute name="title">
             <string>Historical operations</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="myQTableWidget" name="tblInvestmentsHistoricas">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Date</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Years</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Investment</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Operation type</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Initial balance</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Final balance</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Gross</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Comissions</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Taxes</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Net</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>% Total</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>Before</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QTabWidget" name="tabResultados">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_4">
            <attribute name="title">
             <string>Investment situation</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="myQTableWidget" name="tblInvestmentsActualAntes">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Date</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Shares</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Price</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Balance</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Pending</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>% Year to Date</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>% APR</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>% Total</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Benchmark</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">myqtablewidget</header>
  </customwidget>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">myqlineedit</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
